### ISSUE TITLE
Add more Resolution and Framerate options.

### ISSUE THREAD

**Issue by summershrimp** [summershrimp/obs-ssp#34]
Currently we only give 4k/1080p and 25/30/50/60p. It's better to give all resolution and framerate that selected machine can running at.

**Comment by pgannett** [comment]
Could you point me to where in the source code these settings are?

I've got a project I want to stream 3696x2772 from an E2 (0.96 firmware). If you point me in the right direction I could test and let you know how it works.

**Comment by summershrimp** [comment]
https://github.com/summershrimp/obs-ssp/blob/3f925d0ab9aaa69054eb43e8fa23b70c5402740e/src/obs-ssp-source.cpp#L356
Current version is hard-coded resolution and framerate. I'm planning to hard-coded more by model number, or get all supported resolution and framerate  by api.

**Comment by lupupitkanen** [comment]
720p (1280x720) would be great since that's the most common live streaming output resolution and requires the least amount of CPU when decoding.

**Comment by olivier-biron** [comment]
This could be the reason why i can't use this with a V1-Pro.
Would be great to add the resolution for this camera.

**Comment by summershrimp** [comment]
> This could be the reason why i can't use this with a V1-Pro.
> Would be great to add the resolution for this camera.

I don't have V series camera, and I think it would be more complicate for a VR camera to live on obs.

**Comment by summershrimp** [comment]
> At this point i am not looking for anything fancy but simply to pipe the feed.
> That i could give a try to play around with obs prosense stitcher.
> I think i understand where to add the resolution but i am still very bad at compiling etc.
> would it be too much to ask to add 2120x1248 p60 and 3680 x 2428 p30 in your next release :)
> That would be great.

You can try the latest windows version and checked the 'Don't Check Device Status' checkbox on ssp configuration widget. And see if the stream will shown. If it shown I'd ask for your time to having a remote controlling to your computer and check for the http api response of K1

**Comment by olivier-biron** [comment]
Yes the plug in sense the 9 streams and the api is based on the one from the E1 - K1 but i know there is some exception here and there. Kind of a hybrid between E1 and E2
